scripts/operator.js

```diff
@@ -538,27 +539,45 @@ function Operator(el)
       var type = file.type;
 
       if (type === 'image/jpeg' || type === 'image/png' || type === 'image/gif') {
```

Comment:
I'd kinda like to see svg drag/drop and paste support here too. Is it possible or nah? I haven't experimented with them enough.